randovania/games/dread/json_data/Artaria.txt

```diff
@@ -3757,7 +3757,7 @@ Extra - asset_id: collision_camera_056
           Flash Shift or Movement (Advanced) or Use Spin Boost
           Speed Booster and Disabled Door Lock Randomizer
           Movement (Beginner) and Lay Cross Bomb
-          Infinite Bomb Jump (Advanced) and Lay Normal Bomb
+          Diagonal Bomb Jump (Beginner) and Lay Normal Bomb
```

Comment:
i'd definitely not mark this as beginner, cause it involves going mostly sideways with little vertical movement